bin/cylc-insert

```diff
@@ -43,39 +43,47 @@ if '--use-ssh' in sys.argv[1:]:
 import cylc.flags
 from cylc.prompt import prompt
 from cylc.network.suite_command import SuiteCommandClient
-from cylc.CylcOptionParsers import cop, multitask_usage
+from cylc.CylcOptionParsers import cop
 
 
 def main():
     parser = cop(
-        __doc__ + multitask_usage,
-        pyro=True, multitask=True,
+        __doc__, pyro=True, multitask=True,
         argdoc=[
             ("REG", "Suite name"),
-            ('MATCH', 'Task or family name matching regular expression'),
-            ('CYCLE_POINT', 'Cycle point (e.g. date-time or integer)'),
-            ('[STOP_POINT]', 'Optional stop cycle point for inserted task.')])
```

Comment:
It has not been removed. It is now the `--hold-point=POINT` option. (The old interface is also fully supported In backward compatibility mode.)